client/build.gradle

```diff
@@ -32,3 +32,5 @@ apply from: generateDescriptorSetScript
 apply from: testArtifactsScript
 
 apply plugin: spineProtobufPluginId
+
+apply plugin: spineJavadocFqnCheckerId
```

Comment:
Let's apply the plugin in `subprojects` section in the root project instead of applying it in many files.